src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/entity/AssessmentStatus.kt

```diff
@@ -40,12 +40,6 @@ enum class AssessmentStatus {
         TaskProgress.Fixed(PRINT_LICENCE, LOCKED),
       ),
     )
-
-    override val visibleToRole = mapOf(
```

Comment:
Generally all cases will be viewable to a given role, it's just the tab that they appear on that differs.